src/components/graph/graph.ts

```diff
@@ -314,10 +315,26 @@ export default class GraphComponent {
 
   // @ts-ignore // no unused variable
   private _enableClickToCenter(): void {
+    let el: any = null
```

Comment:
We don't want to do any of this. It's do nothing but cause issues later on and make the code harder to reason about.